packages/app/src/messages/components/messages/messages.component.tsx

```diff
@@ -134,6 +98,8 @@ const Messages: FunctionComponent<Props> = ({
   currentlyDeletingMessageId,
   resendMessage,
 }) => {
+  // TODO move component logic to custom hook
```

Comment:
Maybe we should add a task for that? Otherwise there is a chance it would stay here for a looong time :D

(Or feel free to ignore it if you have any other plan for that)

Reply:
https://github.com/mudita/mudita-center/pull/815/commits/43aa30a30b9e165b993db7522287ef4b4447a049